docs/static/spec/rasa.yml

```diff
@@ -1214,7 +1214,11 @@ components:
           description: Type of the entity
         confidence:
           type: number
-      required: ["start", "end", "value", "entity"]
+	      additional_info:
+	        type: object
+	      text:
+	        type: string
```

Comment:
Lines 1217 and L1219 also seem over-indented and would fail yaml parsing, did you mean these are additional keys belonging to `confidence` key?